### PR TITLE
fix: Clean up sign-in section of quick start guide

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

I think this paragraph is a bit outdated. Aligned it with https://www.cloudquery.io/download.
`cloudquery login` also handles creating an account (via the browser Window) so we don't need that part either.

Feedback welcomed

### CODE REVIEW

**Review by JoeKarlsson — APPROVED** [review]
LGTM

File: website/components/mdx/_cloudquery-sign-in.mdx [hunk 1]
```diff
@@ -21,12 +21,6 @@ export const SignIn = ({ os }) => {
 {/* <!-- vale on --> */}
 ## Sign in with CloudQuery
 
-We recommend that you authenticate using the CloudQuery CLI. Once signed in, you can use CloudQuery with premium plugins up to a free quota and premium plugins in preview without limitations.
-
-Authenticated users also have higher rate limits on plugin downloads than anonymous users.
-
-You will be able to create a new account when you sign in with the CLI for the first time. Alternatively, head to [CloudQuery Cloud](https://cloud.cloudquery.io) to create a new account.
```
[the remark this answers is not in the document]
Comment (pilvikala) [reply website/components/mdx/_cloudquery-sign-in.mdx:28]: The paragraph makes it sounds like logging in is optional but it really isn't except a few cases. I'd leave it there, but I'd rephrase it a little. Just login, create an account if you haven't done so yet.

[the remark this answers is not in the document]
Comment (erezrokah) [reply website/components/mdx/_cloudquery-sign-in.mdx:28]: Sorry for the late follow up got caught up with other things, let me know if this works https://github.com/cloudquery/cloudquery/pull/19102/commits/1dad0a2ab4bed7d92025399fedf5629c8c23985d

File: website/components/mdx/_cloudquery-sign-in.mdx [hunk 2]
```diff
@@ -21,17 +21,12 @@ export const SignIn = ({ os }) => {
...
-
 To sign in from the CLI, run the following command:
 
 <SignIn os={props.os} />
 
 A new browser window will open where you will complete the sign-in process.
+You will be able to create a new account when you sign in with the CLI for the first time. Alternatively, head to [CloudQuery Cloud](https://cloud.cloudquery.io) to create a new account.
```
Comment (JoeKarlsson) [inline website/components/mdx/_cloudquery-sign-in.mdx:29]:
```suggestion

You will be able to create a new account when you sign in with the CLI for the first time. Alternatively, head to [CloudQuery Cloud](https://cloud.cloudquery.io) to create a new account.
```

**Review by JoeKarlsson — APPROVED** [review]

**Review by hermanschaaf — APPROVED** [review]
